### DISCUSSION TITLE
Polyfilling globalThis

### DISCUSSION THREAD

**Discussion by pieterdd** [sveltejs/kit#13013] [Q&A] [answered]
I'd like to target older versions of Safari and noticed `globalThis` needs a polyfill. What's the recommended way to do this? I played with Vite's plugin-legacy, but I know there are some [incompatibilities](https://github.com/sveltejs/kit/issues/9515).

**Comment by deleteme** [comment, marked as answer]
```
if (!window.globalThis) window.globalThis = window;
```

**Reply by pieterdd** [reply]
Aha... in the browsers I tested `globalThis` simply appears to be an alias for `window`, so that actually makes a lot of sense. Thanks!
